Dockerfile

```diff
@@ -2,16 +2,17 @@ FROM openjdk:8u212-jre
 
 WORKDIR /opt
 
-ENV HADOOP_HOME=/opt/hadoop-2.9.2
-ENV HIVE_HOME=/opt/apache-hive-2.3.5-bin
+ENV HADOOP_HOME=/opt/hadoop-3.1.3
+ENV HIVE_HOME=/opt/apache-hive-metastore-3.0.0-bin
 # Include additional jars
-ENV HADOOP_CLASSPATH=/opt/hadoop-2.9.2/share/hadoop/tools/lib/aws-java-sdk-bundle-1.11.199.jar:/opt/hadoop-2.9.2/share/hadoop/tools/lib/hadoop-aws-2.9.2.jar
+ENV HADOOP_CLASSPATH=/opt/hadoop-3.1.3/share/hadoop/tools/lib/aws-java-sdk-bundle-1.11.271.jar:/opt/hadoop-3.1.3/share/hadoop/tools/lib/hadoop-aws-3.1.3.jar
```

Comment:
Does it make sense to do a replace here of `/opt/hadoop-3.1.3` with `${HADOOP_HOME}`?

Reply:
Yes. it would be great PR to improve this!